sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/aio/_client.py

```diff
@@ -58,6 +58,7 @@ async def create_key(self, name: str, key_type: "Union[str, KeyType]", **kwargs:
         :keyword curve: Elliptic curve name. Applies only to elliptic curve keys. Defaults to the NIST P-256
          elliptic curve. To create an elliptic curve key, consider using :func:`create_ec_key` instead.
         :paramtype curve: ~azure.keyvault.keys.KeyCurveName or str
+        :keyword int public_exponent: The RSA public exponent to use. Applies only to RSA keys created in a Managed HSM.
```

Comment:
We should document this on `create_rsa_key` as well, no?

Reply:
We should! I remember making that update in a dream last night, but I evidently didn't remember to translate that into the material world